src/Sql/dbo/Stored Procedures/User_Update.sql

```diff
@@ -37,7 +37,11 @@
     @FailedLoginCount INT,
     @LastFailedLoginDate DATETIME2(7),
     @UnknownDeviceVerificationEnabled BIT = 1,
-    @AvatarColor VARCHAR(7)
+    @AvatarColor VARCHAR(7),
```

Comment:
⚠️ Not in scope here but this should have a default. All parameters should really ...

Reply:
We've actually covered this particular miss in our retro from last release; it wouldn't have been backward compatible if we had issues during that release.  Would you like me to create a migration just to add the default?  No problem if you'd like it.

Reply:
@trmartin4 if we can spare a little time, I think it's worth us looking at all procs without defaults and fixing this en masse. Good step towards being backward-compatible 100%. Unsure if that's worth a migration script to apply vs. just change the master copy though; I figure eventually it'd trickle into future updates.